app/views/curation_concerns/base/_form_required_information.html.erb

```diff
@@ -1,3 +1,7 @@
+<% unless params[:parent_id].blank? %>
+  <%= f.hidden_field :parent_id, value: params[:parent_id] %>
```

Comment:
Looks like it's using /concern/scanned_resources/ instead of /container/:parent/scanned_resources/ -- but I can fix that instead of just inserting the hidden parent_id field.